test/cases/wasm/js-incompatible-type/index.js

```diff
@@ -0,0 +1,11 @@
+it("should disallow exporting a func signature with result i64", function() {
+    return expect(import("./export-i64-result.wat")).rejects.toThrow(/invalid type/);
```

Comment:
I don't think this should throw because there could be other usable exports and we don't know which exports are used here.